### PR TITLE
Update api-server rules

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Update rules and enable for ocp4

### CODE REVIEW

File: applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Configure the Encryption Provider Cipher'
+
+description: |-
+    To configure OpenShift to use the <tt>aescbc</tt> encryption provider,
+    follow the OpenShift documentation and configure a
+    <tt>EncryptionConfig</tt> file <tt>/etc/origin/master/encryption-config.yaml</tt>.
```
Comment (shawndwells) [inline applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml]:
```suggestion
    <tt>EncryptionConfig</tt> file at <tt>/etc/origin/master/encryption-config.yaml</tt>.
```

File: applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Configure the Encryption Provider Cipher'
+
+description: |-
+    To configure OpenShift to use the <tt>aescbc</tt> encryption provider,
+    follow the OpenShift documentation and configure a
```
Comment (shawndwells) [inline applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml]:
```suggestion
    follow the OpenShift documentation and create or modify  a
```

File: applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <tt>EncryptionConfig</tt> file <tt>/etc/origin/master/encryption-config.yaml</tt>.
+    In this file, choose <tt>aescbc</tt> as the encryption provider:
+    <pre>kind: EncryptionConfig
+    apiVersion: v1
+    resources:
+      - resources:
+        - secrets
```
Comment (shawndwells) [inline applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml]: Are these lines related to this configuration setting?

`````
      - resources:
        - secrets
`````

If not, should they be removed from the example? Some users may literally interpret these lines and assume they must be present and blank for this check.

File: applications/openshift/api-server/api_server_encryption_provider_cipher/oval/shared.xml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    <criteria operator="AND">
+      <criterion comment="encryption policy files exists" test_ref="test_encryption_policy_file_exists" />
+      <criterion comment="aescbc cipher is configured" test_ref="test_api_server_experimental_encryption_provider_cipher" />
+    </criteria>
+  </definition>
+
+  <unix:file_test check="all" check_existence="all_exist" comment="encryption policy files exists" id="test_encryption_policy_file_exists" version="1">
+    <unix:object object_ref="object_encryption_policy_file_exists" />
+  </unix:file_test>
+  <unix:file_object comment="/etc/shadow" id="object_encryption_policy_file_exists" version="1">
```
[the remark this answers is not in the document]
Comment (redhatrises) [reply applications/openshift/api-server/api_server_encryption_provider_cipher/oval/shared.xml]: @shawndwells will be updating the OVAL is follow-on PR because OVAL currently doesn't handle JSON and such.

File: applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml (hunk 3 above)
Comment (redhatrises) [reply applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml]: @shawndwells can't say yet. The example will be changing for JSON.
